### PR TITLE
Update use-the-link-card.md

### PR DESCRIPTION
**Pull request by lucabandMSFT**
added Sway entry in the table at the bottom of the doc.

### CODE REVIEW

File: Viva/connections/use-the-link-card.md [hunk 1]
```diff
@@ -151,6 +151,8 @@ The best experiences will be achieved if the service does not require authentica
 | Power Portals  | Power portals opens within Teams, user is asked to sign-in on the first time and user doesn’t need to authenticate again if they stay signed in.  | Opens a web browser session and the user might need to re-authenticate depending on browser and machine settings.  |
 | Stream   | Stream opens within Teams, user is asked to sign-in on the first time and user doesn’t need to authenticate again if they stay signed in.   | Opens a web browser session and the user might need to re-authenticate depending on browser and machine settings. |
 | External Links  | Web view opens within Teams and the user might need to authenticate again (depending on the site.)  | Opens a web browser session and the user might need to re-authenticate depending on browser and machine settings.  |
+| Sway  | Sway opens within Teams, user is asked to sign-in on the first time, and user doesn’t need to authenticate again if they stay signed in. | Sway opens within Teams, user is asked to sign-in on the first time, and user doesn’t need to authenticate again if they stay signed in.            |
```
Comment (JohanFreelancer9) [inline Viva/connections/use-the-link-card.md:154]:
```suggestion
| Sway  | Sway opens within Teams and the user is asked to sign in the first time. The user doesn’t need to authenticate again if they stay signed in. | Sway opens within Teams and the user is asked to sign in the first time. The user doesn’t need to authenticate again if they stay signed in.            |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @lucabandMSFT, kindly commit the suggested changes. Many thanks.

**Review by lucabandMSFT — COMMENTED** [review]
@JohanFreelancer9 , looks like I can't approve here. Is there anything I have to do?

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by JohanFreelancer9** [comment]
> @JohanFreelancer9 , looks like I can't approve here. Is there anything I have to do?

@lucabandMSFT, nothing more is needed from you at this time. Many thanks.

**Comment by scanum** [comment]
@Holland-ODSP 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by Holland-ODSP — APPROVED** [review]

**Pull request merged**